scRNA-seq/03-normalizing_scRNA.Rmd

```diff
@@ -197,15 +202,38 @@ Feel free to customize the plot with a different theme or color scheme!
 Let's plot the `norm_pca_scores` data:
 
 ```{r norm_pca_plot, live = TRUE}
+ggplot(norm_pca_scores, aes(x = PC1, y = PC2, color = total_umi)) +
+  geom_point() +
+  labs(title = "Normalized log counts PCA scores",
+       color = "Total UMI count") +
```

Comment:
```suggestion
       color = "Total UMI count") +
       scale_color_viridis_c()
```
Can we add a more distinct palette? The blue scales are hard to distinguish other than really light points. I want to make sure that if we choose to show this it's actually showing that unnormalized scores means most of the variation may come from total UMI count. If that isn't obvious, then we shouldn't add this in.

Reply:
I'll repost with how this looks !